templates/login_or_signup.tmpl

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <meta
+            name="description"
+            content="The best way to book club" />
+        <meta
+            name="viewport"
+            content="width=device-width, initial-scale=1" />
```

Comment:
what about this one?

Reply:
I understand this one less, but it has something to do with setting the width of the page. See: https://css-tricks.com/snippets/html/responsive-meta-tag/